### ISSUE TITLE
Package validation and interface inheritance problem

### ISSUE THREAD

**Issue by hudecekd** [dotnet/sdk#25888]
# Problem
Greetings,

we use package validation which is great and currently we are facing issue (maybe it is feature).
We have extended our interface from `IBackgroundTaskQueue` to `IBackgroundTaskQueue : IDisposable`.
I would expect that when we try to make package it shows error since we introduced new required metho `Dispose` which has
to be implemented in class which implements this interface.
But there is no error. It seems odd to me since if I add some method to interface directly it shows error such as
```
error CP0006: Cannot add interface member 'IBackgroundTaskQueue.Test()' to lib/netstandard2.1/XXX.dll because it does not exist on [Baseline] lib/netstandard2.1/XXX.dll
```

Thanks for advice
Dusan Hudecek

# Current behavior
No error package builds and package validation reports NO error.

# Expectation
I would expect that error similar to CP0006 is shown.

# sdk
.NET 6

**Comment by ViktorHofer** [comment]
@marcpopMSFT can we please configure a repo bot to get notified for area-compatibility issues? I just saw this because of coincidence.

**Comment by marcpopMSFT** [comment]
@ViktorHofer , it might be worth going back through items labeled for compatibility to make sure nothing else was missed. Do you know how other repos are doing notifications? We mostly just rely on the labeler and I wonder if the labeler can be configured to send notifications as well.

@danmoseley do you happen to know?

**Comment by danmoseley** [comment]
>  I wonder if the labeler can be configured to send notifications as well.

It does not (cc @Eilon in case he knows otherwise). In dotnet/runtime, we use the Fabricbot for notifications. When the labeler assigns an area, Fabricbot sees the event and tags the area. @jeffhandley controls this and can suggest how you might do it. He has an elaborate json file configuring it, but if you only want to send notifications for one area, maybe something simpler would be sufficient.

**Comment by Eilon** [comment]
Yup, FabricBot is the way to go. Very easy to set up for something like this.

**Comment by marcpopMSFT** [comment]
@Eilon do you have pointers to the fabric bot documentation? I think this will potentially work but what I don't know is how to limit it to only new issues.  Any suggestions?
https://github.com/dotnet/sdk/pull/25959

**Comment by Eilon** [comment]
Is there a reason you want to limit to only new issues? Or would you want all issues, upon being tagged as compatibility, to alert a team/user? (I would think you'd always want it, no?)

You can find some FabricBot info via this little readme I wrote a while ago: https://github.com/dotnet/maui/blob/main/.github/fabricbot_readme.md. But official docs are somewhat limited.

**Comment by marcpopMSFT** [comment]
@Eilon , how do I keep the fabricbot from making the same comment every day?  That's why I wanted only new issues as then it wouldn't comment on the same issue again and again which I'm pretty sure my PR will do. Do I need to label it and then have the bot exclude that label? What about only commenting on untriaged items?

**Comment by Eilon** [comment]
You would set the rule to trigger _when_ the compatibility rule label is added, so it should happen exactly once.

Something like this:

<img width="1173" alt="image" src="https://user-images.githubusercontent.com/202643/174881445-80b48e04-a95e-47da-b1da-56d2c67fecd7.png">

So this should happen exactly one time, unless someone removes the label, adds it, repeat, etc. (but then tagging the team multiple times actually seems reasonable to me).

**Comment by hudecekd** [comment]
Hi @Eilon ,
shouldn't your discussion have its own issue? it seems that all comments now are not about the original problem but about Fabricbot and it can make issue comments not clear since they are not related to issue.
Thanks

**Comment by Eilon** [comment]
@hudecekd you know what, I never even noticed what the original issue here was because I was only responding to individual comments 😄 Anyway, I logged https://github.com/dotnet/sdk/issues/26166 to track creating automation rules, and I've hidden the irrelevant comments from this issue. Thanks for keeping us honest 😁
